icu4c/source/test/cintltst/ucptrietest.c

```diff
@@ -930,17 +998,15 @@ testTrieSerializeAllValueWidth(const char *testName,
 static UMutableCPTrie *
 makeTrieWithRanges(const char *testName, UBool withClone,
                    const SetRange setRanges[], int32_t countSetRanges,
-                   const CheckRange checkRanges[], int32_t countCheckRanges) {
+                   uint32_t initialValue, uint32_t errorValue) {
```

Comment:
I am curious -- it's ok to pass less data into this function, but it seems to just complicate the two call sites a little, so nothing really gained?

Reply:
I did this change because it makes more clear that `setRanges` is input to the build step, whereas `checkRanges` is input to the test step.